public/lessons/complexity-analysis/mcq-2.mdx

```diff
@@ -0,0 +1,22 @@
+
+<MCQBlock question="What is the Big-O runtime of this function?
+```c++
+    bool isPalendrome(string str)
+‎   int n = str.length()
+‎   for (int i = 0; i < n / 2; i++)
+‎       if (str[i] != str[n - i - 1])
+‎           return false;
+‎       }
+‎   }
+‎   return true;
+}
+```"
+  answerOptions={[
+    '$O(1)$',
+    '$O(\\frac{n}{2})$',
+    '$O(n)$',
+  ]}
+  optionVerdicts={[false, false, true]}
+  hint="Only consider the worst case scenario. (the string is a palendrome)\ Be sure to drop any constants terms in your result."
```

Comment:
nit: Try considering the worst-case scenario, such as when the string is a palindrome.

(or at least remove the period, it just feels weird next to the parentheses)